components/brave_shields/browser/brave_shields_util.h

```diff
@@ -39,12 +41,14 @@ void SetBraveShieldsEnabled(Profile* profile, bool enable, const GURL& url);
 // reset to the default value
 void ResetBraveShieldsEnabled(Profile* profile, const GURL& url);
```

Comment:
probably combining this bunch of functions into a class could make things simpler

Reply:
they would probably go in BraveShieldsService once we combine with the new BraveShieldsPrefService and convert to a BCKS. Until then I think we might as well leave them here